client/pages/students/[student_id]/index.tsx

```diff
@@ -16,10 +17,13 @@ const ViewStudentPage = () => {
 
   return (
     <div>
-      <h1>Student {student?.student_id}</h1>
+      <h1>Student: {student?.student_id}</h1>
       <p>
         {student?.first_name} {student?.last_name}
       </p>
+      <button>
```

Comment:
`next/link` doesn't need to be wrapped in a button, it renders a `<a/>`

Reply:
I changed it to a <div> for the time being.